sympy/polys/domains/algebraicfield.py

```diff
@@ -124,3 +126,8 @@ def numer(self, a):
     def denom(self, a):
         """Returns denominator of ``a``. """
         return self.one
+
+    def from_AlgebraicField(K1, a, K0):
+        """Convert a polynomial to ``dtype``. """
+        a = K0.to_sympy(a)
+        return K1.from_sympy(a)
```

Comment:
It is not necessary to save `K0.to_sympy(a)` temporarily to a local variable, `K1.from_sympy(K0.to_sympy(a))` should suffice.

Reply:
Yes I agree I will update it.